mh/pat.py

```diff
@@ -806,9 +806,9 @@ def sendAnsServerTime(self, converted_country_code, seq):
         JP: サーバ時刻返答
         TR: Server time response
         """
-        dummy_1 = 0  # in-game time?
-        current_time = time_utils.datetime_to_int(datetime.now())
-        data = struct.pack(">II", dummy_1, current_time)
+        server_time = time_utils.current_server_time()  # Epoch time, counts forward at 1 per second
```

Comment:
AFAIK, we don't have such evidence that the "server_time" is based on epoch so I don't think such comment is appropriate. However, we do know that the `current_time` used for Wii Shop's ticket is using epoch.

![image](https://user-images.githubusercontent.com/7890055/193475009-d391eff6-2a44-41ed-9595-cc4987dc3cd9.png)

Reply:
Changed the comment